communitiesuk/funding-service-design-post-award-data-store
# [SMD-170] round 4 template update db with new columns and migrate

https://dluhcdigital.atlassian.net/browse/SMD-170


### Change description
- adds new round 4 columns to the mappings
- adds new round 4 columns to the data model
- includes a migration file to add new columns 



### How to test

- Migrating your local postgres to the new update
- Manually testing ingest with a Round 3 and Round 4 file to check it works

### Screenshots of UI changes (if applicable)
